### DISCUSSION TITLE
Input field not working when i add value attribute

### DISCUSSION THREAD

**Discussion by theobiabo** [chakra-ui/chakra-ui#6745] [Q&A]
<Input placeholder='Basic usage' value={'item'}/> is not editable 

but it works fine when I remove the value attribute 
![Screenshot from 2022-09-23 12-15-50](https://user-images.githubusercontent.com/54102389/191949147-d018ec99-6625-4438-af4c-66ee2ea4aa84.png)

**Comment by ptim** [comment]
By setting `value`, you're making this a controlled component (intended to be managed by your own `useState` logic or similar, which is not included in your example), and you likely see the following warning in the console:

> react_devtools_backend.js:4012 Warning: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.

Try this:

```jsx
<Input placeholder="Basic usage" defaultValue={'item'} />
```

About controlled vs uncontrolled components: https://reactjs.org/docs/forms.html
